kmapper/kmapper.py

```diff
@@ -498,7 +499,10 @@ def map(
         # Prefix'ing the data with an ID column
         ids = np.array([x for x in range(lens.shape[0])])
         lens = np.c_[ids, lens]
-        X = np.c_[ids, X]
+        if type(X) == csr.csr_matrix:
```

Comment:
Could you please add an example in the docstring above and mention in the Parameters section that `csr_matrix` is a valid format?